example/mixed-arch/main.tf

```diff
@@ -2,22 +2,22 @@ terraform {
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = "=2.51.0"
+      version = ">=2.51.0"
     }
     random = {
       source  = "hashicorp/random"
-      version = "=2.3.0"
+      version = ">=2.3.0"
     }
     kubernetes = {
       source  = "hashicorp/kubernetes"
-      version = "=2.0.2"
+      version = ">=2.0.2"
     }
     helm = {
       source  = "hashicorp/helm"
-      version = "=2.0.3"
+      version = ">=2.0.3"
```

Comment:
Same as last comment, replace operator with `~>` and drop last bit of version.

Reply:
Good catch, updated.